gulp/tasks/scripts.js

```diff
@@ -212,6 +217,13 @@ function copySettings() {
     .pipe(gulp.dest(config.dest));
 }
 
+function copySettings() {
```

Comment:
This doesn't seem to be working right. Loading the page gives me 'Cannot read property `apiOrigin` of undefined'. Master branch does not have that problem.

Reply:
You seem to have made a duplicate of the function, and I don't see a difference between them.